README.md

```diff
@@ -89,6 +89,49 @@ You can optionally retrieve all the versions:
 $versions = $entity->versions();
 ```
 
+### Storing Additional MetaData
+
+`cakephp-version` dispatches an event `Model.Version.beforeSave` which you can optionally handle to attach additional meta-data about the version.
```

Comment:
Can you show how someone might be able attach the listener (just for clarity's sake).